packages/shared/forks/ReactFeatureFlags.www-dynamic.js

```diff
@@ -36,16 +34,6 @@ export const enableDebugTracing = __EXPERIMENTAL__;
 
 export const enableSchedulingProfiler = __VARIANT__;
 
-// This only has an effect in the new reconciler. But also, the new reconciler
-// is only enabled when __VARIANT__ is true. So this is set to the opposite of
-// __VARIANT__ so that it's `false` when running against the new reconciler.
-// Ideally we would test both against the new reconciler, but until then, we
-// should test the value that is used in www. Which is `false`.
-//
-// Once Lanes has landed in both reconciler forks, we'll get coverage of
-// both branches.
-export const deferRenderPhaseUpdateToNextBatch = !__VARIANT__;
```

Comment:
I hesitated a bit with this `!`. From what I could tell this is just affecting how the value is set in the test run and not inlined as `! dynamicFlags.theFlag`.

So I hardcoded this value as `false`.